pkg/apis/options/options.go

```diff
@@ -2,13 +2,11 @@ package options
 
 import (
 	"crypto"
-	"net/url"
-	"regexp"
-
 	oidc "github.com/coreos/go-oidc"
 	ipapi "github.com/oauth2-proxy/oauth2-proxy/pkg/apis/ip"
 	"github.com/oauth2-proxy/oauth2-proxy/providers"
 	"github.com/spf13/pflag"
+	"net/url"
```

Comment:
This should be back where it was up top with the standard library imports separated